backend/app/Http/Controllers/User/ShowEditUserController.php

```diff
@@ -93,6 +109,28 @@ public function __invoke(): View|Factory
             'name' => $id->label(), //学部名
         ], $uuFacultyEnum);
 
+        /**
+         * ユーザー情報追加取得
+         */
+        //学科
+        /** @var UUMajor */
+        $uuMajor = UUMajor::where('id', $userInfo->u_u_major_id)->first();
+        $userInfo->u_u_faculty_id = $uuMajor->uu_faculty_id;
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Access to an undefined property App\Models\UserInfo::$u_u_faculty_id.